pomdp_py/problems/cc_rocksample/cc_rocksample_problem.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import annotations
+import copy
+import math
+import pomdp_py
+from pomdp_py.problems.rocksample.rocksample_problem import (
+    RockSampleProblem,
+    create_instance,
+    RSTransitionModel,
+    RSObservationModel,
+    RSPolicyModel,
+    CheckAction,
+    RSRewardModel,
+    init_particles_belief,
+    State,
+)
+
+
```

Comment:
Would be really nice to have some other domains; Rocksample is really old and kind of detached from reality. If cost-constraint is really useful it should find applications in other more realistic domains.